lib/mito/io/public.h

```diff
@@ -0,0 +1,17 @@
+// code guard
+#if !defined(mito_io_public_h)
+#define mito_io_public_h
+
+
+// external packages
+#include "externals.h"
+
+// classes implementation
+#include "mesh/reader.h"
+#ifdef WITH_VTK
+#include "mesh/writer.h"
```

Comment:
I would move the `writer.h `into a `vtk` subdirectory of `lib/mito/io`, so this piece of code would read

```
#ifdef WITH_VTK
#include "vtk/public.h"
#endif
```

Then file `public.h` would include file `writer.h`.

Reply:
Done